CONTRIBUTING.md

```diff
@@ -70,138 +70,61 @@ You have a cool idea, but do not know know if it fits with Code Carbon? You can
 <!-- TOC --><a name="installation"></a>
 ### Installation
 
-CodeCarbon is a Python package, to contribute to it, you need to have Python installed on your machine, natively or with [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/).
+CodeCarbon is a Python package, to contribute to it, you need to have Python installed on your machine, natively or with [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/), or better, faster, stronger with [UV](https://github.com/astral-sh/uv).
 
-Since April 2024 we use Hatch for managing development environment. Hatch manage the environments, even the Python versions, the dependencies and handle matrix testing. It is a good way to avoid conflicts with your system Python.
+Between April 2024 and July 2025 we use Hatch for managing development environment. Since August 2025 we use UV manages the environments, Python versions, and dependencies - it's a fast, reliable way to work with Python projects.
 
 We have dropped support of Python 3.6 since version 2.0.0 of CodeCarbon.
 
-It is not mandatory for small contribution, while not recommanded, you could just install the package with `pip install -e .`.
+Please install [UV](https://github.com/astral-sh/uv) following [installation instructions](https://github.com/astral-sh/uv#installation), or with:
 
-Please install [Hatch](https://hatch.pypa.io/) following [installation instruction](https://hatch.pypa.io/latest/install/), or with `pipx install hatch hatch-pip-compile`.
+```sh
+pip install uv
```

Comment:
Maybe suggest a python less uv install, following the documentation ? https://docs.astral.sh/uv/getting-started/installation/#standalone-installer

Reply:
OK, I've done it.